spec/features/providers/allocations_spec.rb

```diff
@@ -61,6 +61,28 @@
     and_i_see_the_success_page
   end
 
+  scenario "Accredited body declines request for PE allocations" do
+    given_accredited_body_exists
+    given_training_provider_with_pe_fee_founded_course_exists
+
+    given_i_am_signed_in_as_an_admin
+
+    when_i_visit_my_organisations_page
+    and_i_click_request_pe_courses
+    then_i_see_the_pe_allocations_page
+
+    and_i_see_only_see_training_providers_offering_pe_fee_founded_courses
```

Comment:
then

Reply:
i will leave the line as is and i will remove the empty line above as i think it flows on